daprdocs/content/en/reference/api/jobs_api.md

```diff
@@ -63,6 +64,39 @@ Entry                  | Description                                | Equivalent
 @daily (or @midnight)  | Run once a day, midnight                   | 0 0 0 * * *
 @hourly                | Run once an hour, beginning of hour        | 0 0 * * * *
 
+#### failure_policy
+
+`failure_policy` specifies how the job should handle failures.
+
+It can be set to `constant` or `drop`.
+- The `constant` policy retries the job based on the configuration.
+  - `max_retries` configures how many times the job should be retried. Not setting this makes it retry indefinitely.
+  - `interval` configures the delay between retries. Not setting this makes it retry immediately.
```

Comment:
```suggestion
  - `interval` configures the delay between retries. Defaults to retrying immediately. Valid values are of the form `200ms`, `15s`, `2m`, etc.
```

Reply:
Not sure if all these are valid but this is what we say for other retry policies: https://docs.dapr.io/operations/resiliency/policies/retries/retries-overview/#spec-metadata